packages/devtools_app/lib/src/split.dart

```diff
@@ -88,15 +95,36 @@ class Split extends StatefulWidget {
 
 class _SplitState extends State<Split> {
   List<double> fractions;
+  List<double> fractionsStorage;
+  ValueListenable<List<double>> _fractionsStorage;
 
   bool get isHorizontal => widget.axis == Axis.horizontal;
 
   @override
   void initState() {
     super.initState();
+    if (widget.fractionsKey != null) {
+      _fractionsStorage = preferences.lookupSplitFractions(widget.fractionsKey);
+      fractionsStorage = _fractionsStorage.value;
+      _fractionsStorage.addListener(_onChangeSplitFractions);
```

Comment:
instead of manually adding and removing the listener, use the `addAutoDisposeListener` helper that is used throughout devtools